repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -705,6 +730,82 @@ export const getPassengerTypeByNameAndNocCode = async (
     }
 };
 
+export const getSinglePassengerTypeByNameAndNationalOperatorCode = async (
```

Comment:
What's the difference this and getPassengerTypeByNameAndNocCode ?

Reply:
This one returns `SinglePassengerType` which has the following structure:

```
export interface SinglePassengerType {
  id: number;
  name: string;
  passengerType: PassengerType;
}
```

`getPassengerTypeByNameAndNocCode` returns `PassengerType`.